packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -54,6 +54,7 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
     protected waitForConnection: Deferred<MessageConnection> | undefined;
     protected hoverMessage: HTMLDivElement;
     protected lastTouchEnd: TouchEvent | undefined;
+    protected loadingMessage: HTMLDivElement;
```

Comment:
Are you considering using some **icon-loader** for terminal tab till connection to the task process is not recovered _instead of_ a **loading message**?
Like here, for example:

![loader-icon](https://user-images.githubusercontent.com/5676062/74089806-cbf74900-4aad-11ea-8cb7-2e4a2871b229.gif)